src/sentry/south_migrations/0457_global_searches_data_migration.py

```diff
@@ -0,0 +1,1304 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import DataMigration
+from django.db import models
+
+from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+
+DEFAULT_SAVED_SEARCHES = [
+    {
+        'name': 'Unresolved Issues',
+        'query': 'is:unresolved',
+        'is_default': True
```

Comment:
did you need this `is_default`? didn't look like it was used -- not sure if this just doesn't apply to the global saved searches